ui/packages/catalog-ui-search/src/main/webapp/component/metacard-interactions/metacard-interactions.view.js

```diff
@@ -75,7 +77,12 @@ define([
             this.checkHasLocation();
             this.setupResultAdd();
             this.setupResultActionsExport();
+            const extensions = this.getExtensions(this.model);
```

Comment:
❓ Why do we pass the model into this when it should be able to access it by calling `this.model` within the method?

Reply:
We could allow the `getExtensions` method to dig around it's `this` context, however if we pass it in, we are defining an explicit contract. I am not sure which is better.